gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/DbDataStoreTests.js

```diff
@@ -746,39 +808,49 @@ fluid.defaults("gpii.tests.dbDataStore.findAuthorizedClientsByGpiiToken", {
     }]
 });
 
-fluid.defaults("gpii.tests.dbDataStore.findAuthByAccessToken", {
+fluid.defaults("gpii.tests.dbDataStore.findAuthorizationByAccessToken", {
```

Comment:
This file has very good coverage but could we add one more test: finding a GPII App Installation Authorization by Access Token?

Reply:
In this pull request, this function only deals with web prefs consumers and onboraded solutions because it's an API used by [authGrantFinder.getGrantForAccessToken](https://github.com/cindyli/universal/blob/GPII-2435/gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthGrantFinder.js#L45) and in this pull request, the output of `getGrantForAccessToken` for gpii app installations is not clear yet. This function is enhanced to deal with gpii app installations in [the GPII-2538 branch](https://github.com/cindyli/universal/blob/GPII-2538/gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/dbViews/views.json#L48).

Reply:
Ok, this is reasonable.